clang/bindings/python/clang/cindex.py

```diff
@@ -2482,6 +2529,13 @@ def spelling(self):
     FLOAT128 = 30
     HALF = 31
     FLOAT16 = 32
+    SHORTACCUM = 33
+    ACCUM = 34
+    LONGACCUM = 35
+    USHORTACCUM = 36
+    UACCUM = 37
+    ULONGACCUM = 38
+    BFLOAT16 = 39
     IBM128 = 40
```

Comment:
I don't see constants that correspond to `CXType_FirstBuiltin` and `CXType_LastBuiltin`. Is this intentional?

Reply:
They're aliases and at least not used internally, I'm not sure I understand the point of those.
I just tested that enum aliases are not problem with the Python enum class. 
Note however that we added a test for duplicate enums a while back here: https://github.com/llvm/llvm-project/blob/main/clang/bindings/python/tests/cindex/test_enums.py#L47
which expects no two enums to have the same number. Changing this would break that test (I guess we could compare by exact names instead, which doesn't work if someone changes the spelling when introducing a duplicate though)

Reply:
No, the test does break. But I realized this also makes the test pointless now: the only thing it does is test precisely that we do *not* have aliases. The entire module already throws an error on load if the same variant is redefined.
I added the aliases and removed the test